AlphaWallet/Export/ViewModels/PromptBackupWalletAfterExceedingThresholdViewViewModel.swift

```diff
@@ -16,7 +16,7 @@ struct PromptBackupWalletAfterExceedingThresholdViewViewModel: PromptBackupWalle
     }
 
     var description: String {
-        let prettyAmount = CurrencyFormatter.formatter.string(from: NSNumber(value: dollarValueInUsd)) ?? "-"
+        let prettyAmount = NumberFormatter().currency.string(from: NSNumber(value: dollarValueInUsd)) ?? "-"
```

Comment:
> I'll need to check it somemore, but what if instead of:
> 
> ```
> let prettyAmount = NumberFormatter().currency.string(from: NSNumber(value: dollarValueInUsd)) ?? "-"
> ```
> 
> we make it like this:
> 
> ```
> let prettyAmount = NumberFormatter.currency.string(from: NSNumber(value: dollarValueInUsd)) ?? "-"
> ```
> 
> So we don't need to create an instance of `NumberFormatter` every time, and also we can cache them as static vars? formatters have poor initialization performance and we since we really only have a handful unique ones, it feels like we can make do with static versions.
> 
> The new names and API are definitely more Swifty though. They are good.

@hboon it seems like done, you can check it.